scripts/qa_crowbarsetup.sh

```diff
@@ -3334,6 +3335,7 @@ function deploy_single_proposal()
             ;;
         trove)
             iscloudver 5plus || return
+            [[ $want_trove = 1 ]] || return
```

Comment:
I guess @nicolasbock was asking if it makes more sense to phrase it in a positive way like:
` [[ $want_trove = 0 ]] && return`

Same question may apply to the line above:
`iscloudver 4minus && return`

But thats a matter of taste.